README.old.md

```diff
@@ -0,0 +1,75 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] Kindly make your readme descriptive by adding the neccessary information.
- [ ] Also add a link to your video description in the readme file and the pull request description.

Reply:
Thanks for your review! I forgot, sorry..